src/main/java/org/folio/service/export/storage/AWSStorageServiceImpl.java

```diff
@@ -74,13 +83,32 @@ private Date getExpiration() {
     expTimeMillis += EXPIRATION_TEN_MINUTES;
     LOGGER.info("AWS presigned URL link expiration time millis {}", expTimeMillis);
     expiration.setTime(expTimeMillis);
-
     return expiration;
   }
 
+
   @Override
-  public void storeFile() {
-    // Will be implemented as part of MDEXP-67
+  public void storeFile(FileDefinition fileDefinition, String tenantId) {
+    String folderInS3 = tenantId + "/" + fileDefinition.getJobExecutionId();
+    String bucketName = getProperty(BUCKET_PROP_KEY);
+    if (StringUtils.isNullOrEmpty(bucketName)) {
+      throw new IllegalStateException("S3 bucket name is not defined. Please set the bucket.name system property");
```

Comment:
I think it's better to be consistent in throwing this error, change it here or in the above method

Reply:
Sure.